pkg/virt-api/webhooks/validating-webhook/admitters/validate-k8s-utils.go

```diff
@@ -1,16 +1,21 @@
 /*
-Copyright 2014 The Kubernetes Authors.
```

Comment:
Please consider excluding this file, since its header states it contains code taken from K8s.